subsys/pm/policy/policy_residency.c

```diff
@@ -14,6 +14,36 @@
 #include <logging/log.h>
 LOG_MODULE_DECLARE(pm, CONFIG_PM_LOG_LEVEL);
 
+/**
+ * Check CPU power state consistency.
+ *
+ * @param i Power state index.
+ * @param node_id CPU node identifier.
+ */
+#define CHECK_POWER_STATE_CONSISTENCY(i, node_id)			       \
+	BUILD_ASSERT(							       \
+		DT_PROP_BY_PHANDLE_IDX_OR(node_id, cpu_power_states, i,	       \
+					  min_residency_us, 0U) >	       \
```

Comment:
why not `>=`. What `min_residency_us` == `exit_latency_us` means ?

Reply:
current code asserts for `>`, so I kept it, but yes, makes sense to use `>=`